platforms/hyperledger-besu/configuration/reset-network.yaml

```diff
@@ -110,7 +110,72 @@
         - "{{ network['organizations'] }}"
       ignore_errors: yes
 
-    ##Delete Flux namespace 
+    # Delete ClusterRoleList related to flux
+    - name: Delete flux ClusterRole
+      k8s:
+        state: absent
+        api_version: v1
+        kind: ClusterRole
+        namespace: flux-{{ network.env.type }}
+        name: "{{ item[0] }}"
+        kubeconfig: "{{ kubernetes.config_file }}"
+      vars:
+        kubernetes: "{{ item[1].k8s }}"
+      with_nested:
+        - ["crd-controller-flux-{{ network.env.type }}"]
+        - "{{ network['organizations'] }}"
+      ignore_errors: yes
+
+    # Delete ClusterRoleBinding related to flux
+    - name: Delete flux ClusterRoleBindingList
```

Comment:
Please change it to `ClusterRoleBinding`

Reply:
Hi @jagpreetsinghsasan 

Update to "ClusterRoleBinding" in the latest Pull Request. Thank you.